global-script.lua

```diff
@@ -3125,6 +3273,23 @@ function toggleScoreUI(player)
     colorEnabled = getCurrentState("panelScore", player.color)
     toggleUI("panelScore", player.color, colorEnabled)
 end
+function togglePlayerControls(player)
+    if not player.admin then
```

Comment:
what's the reasoning behind having this with admin controls? so that during the middle of the game someone isn't just constantly swapping colors or moving their spirit around?

Reply:
Basically this, particularly since the button visibility affects _all_ players.  (Unless converted to a UI XML with variable visibility, which is a possible option for future work.)

It also has some utility for -- say -- somebody setting up a teaching game with spirits and colors preselected before other players join.